doc/content/xcp-networkd/host-network-device-ordering.md

```diff
@@ -0,0 +1,342 @@
+---
```

Comment:
Please move the file to `https://github.com/xapi-project/xen-api/tree/master/doc/content/design`

and change the header accordingly

```markdown
---
title: Host Network Device Ordering on Networkd
layout: default
design_doc: true
revision: 1
status: proposed (9.0)
---
```

Reply:
I actually prefer this under xcp-networkd directly as a "how it works" doc rather than a design doc.

Reply:
Yes, but this is a proposal, not how it works now :/